styles.css

```diff
@@ -0,0 +1,723 @@
+body {
+  font-family: 'Montserrat', sans-serif;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 40px 60px 55px 40px;
+  padding-top: 25px;
+}
+
+.header__logo:hover {
+  cursor: pointer;
+}
+
+.menu-model {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 19px;
+  width: 36%;
+}
+
+.menu-model__item {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  position: relative;
+  transition: opacity .5s;
+}
+
+.menu-model__item:first-child {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.menu-model__item:last-child {
+  margin-right: 0;
+}
+
+.menu-model__item_active {
+  color: #e51937;
+}
+
+.menu-model__item::after {
+  content: '';
+  background: url('src/img/underline.svg') bottom no-repeat;
+  display: block;
+  width: 100%;
+  height: 2px;
+  position: absolute;
+  top: 25px;
+  right: 1px;
+  opacity: 0;
+  transition: .5s ease-in;
+}
+
+.menu-model__item:hover {
+  opacity: .5;
+}
+
+.menu-model__item:hover::after, .menu-model__item_active::after {
+  opacity: 1;
+}
+
+.side-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 14%;
+}
+
+.side-menu__item {
+  text-decoration: none;
+  font: 600 15px/18px 'Montserrat', helvetica, sans-serif;
+  color: #000;
+  position: relative;
+}
+
+.side-menu__item::after {
+  content: '';
+  background: url('src/img/underline.svg') bottom no-repeat;
+  display: block;
+  width: 100%;
+  height: 2px;
+  position: absolute;
+  top: 25px;
+  right: 1px;
+  opacity: 0;
+  transition: .5s ease-in;
+}
+
+.side-menu__item:last-child:hover::after {
+  opacity: 0;
+}
+
+.side-menu__item:last-child:hover {
+  opacity: 1;
+}
+
+.side-menu__item:hover {
+  opacity: .5;
+}
+
+.side-menu__item:hover::after {
+  opacity: 1;
+}
+
+.hamburger-menu__icon {
+  width: 24px;
+  height: 15px;
```

Comment:
Codacy found an issue: [Expected selector ".side-menu__item:hover::after" to come before selector ".side-menu__item:last-child:hover::after" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/TESLA-Website-NataSG/pullRequest?prid=6200435)